setup.py

```diff
@@ -11,11 +11,13 @@
 with open(join(this_dir, "requirements.txt")) as f:
     requirements = f.read().split("\n")
 
-requirements.extend(["autoconf==0.9.0", "autoarray==0.21.1", "autofit==0.76.0"])
+version = environ.get("VERSION", "1.0.dev0")

```

Comment:
So we're fixing all the versions of all the projects to be the same? Seems a bit weird. I could imagine a situation when using AutoFit for CTI work where we need to release it independently of AutoLens etc.